readme.md

```diff
@@ -65,6 +71,13 @@ The RGB output format.
 
 Note that when using the `css` format, the value of the alpha channel is rounded to two decimal places.
 
+##### alpha
+
+Type: `number`
+
+The Alpha value to be applied with the first priority.
```

Comment:
This could be better written.

Reply:
I'm sorry for not speaking English well..

I applied your review on cce9864. Please make sure it's okay again